pyist/settings.py

```diff
@@ -150,6 +127,7 @@
     }
 }
 
+# @TODO Should be removed after moving to markdownx package
```

Comment:
markitup'ı `INSTALLED_APPS`'den kaldırdığına göre bunları tutmaya gerek var mı?

Bir de, eski blogları düzgün render etmek için data migration oluşturmaya gerek var mı?

Reply:
Ona baktım, Migration-da kullanıldığı için şimdilik durması gerekiyor, daha sonra merge migration yaparsak gitcek. Şimdilik blog uygulamasının ilk migration-ında duruyor eski kütüphane. Bu 2 parametre olmadan da hata veriyor.

Eski kütüphane `description_rendered` diye field kullanıyordu, o kalktı zaten, yazılar da markdown formatında olduğu için bir değişiklik olmadı (her iki kütüphane render için `markdown` kütüphanesini kullanıyorlar).